R/processes.R

```diff
@@ -0,0 +1,156 @@
+#' @title Define model processes
+#' @description
+#' create_processes, defines the functions which describe how each individual's
+#' states and variables change over time.
+#'
+#' It lists processes from `infection.R`, `mosquito_emergence.R` and
+#' `mortality.R`; and then exposes them to the model through
+#' `bind_process_to_model`
+#' @param individuals, a list of individuals in the model
+#' @param states, a list of states in the model
+#' @param variables, a list of variables and constants in the model
+#' @param parameters, a list of model parameters
+create_processes <- function(individuals, states, variables, parameters) {
```

Comment:
So I can see how the following processes can be used to define progression through infected states, but I can't see how humans get infected in the first place, i.e. where is the process that takes "S" to the latent stage? Again, apologies if I've missed the point here, still getting up to speed with the method.

Reply:
Just read through your original note where you state these things are coming later - ignore the above!